src/coreComponents/common/format/table/TableTypes.hpp

```diff
@@ -38,6 +38,43 @@ enum class CellType : integer
   Hidden
 };
 
+class TableErrorListing
+{
+
+public:
+  /// View on cell error grouping the cell information to display at the end of the table
+  /// Contain all the errors
+  std::vector< string > errorText;
```

Comment:
- Why do you qualify it of a view? It seems more like the "real" list of the error messages (exemples of views: `string_view`, pointer, list of `string_view`)
- I think names like `errorMessages`/`errorsList` would emphasize the possibility to have more than one error stored (it is not a single message with multiple lines). The comment also means that only one error can be stored.
- If the `TableErrorListing` is responsible of the list managment, why is this `std::vector` public?
To keep the possibility of exposing the messages, you could have.
```suggestion
public:
  using Iterator = std::vector< string >::const_iterator;

  Iterator begin()
  { return errorText.begin(); }

  Iterator end()
  { return errorText.end(); }
private:
  std::vector< string > errorText;
```

Reply:
errorText should be private indeed, I will fix that